fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -887,6 +889,8 @@ def encode_list(
                 sequence_id=curr_sequence_id,
             )
             if sub_max_len is not None:
+                if len(sub_encoding) > sub_max_len:
+                    overflow_info += f"{len(sub_encoding)} => {sub_max_len}|"
```

Comment:
I suggest also adding the type of tokenizer being used: overflow_info += f"{input_type}:{len(sub_encoding)} => {sub_max_len}|"

Reply:
This way we'll at least have some idea what tokenizer(s) caused the truncation(s)